File: Python3/inflow_api_get.py
import requests

companyId = ""
apiKey = ""
apiVersion = "2021-04-26"

headers = {
    "Authorization": f"Bearer {apiKey}",
    "Content-Type": "application/json",
    "Accept": f"application/json;version={apiVersion}"
}

url = f"https://cloudapi.inflowinventory.com/{companyId}/sales-orders"

response = requests.get(url, headers=headers)
print(response)

File: TypeScript/inflow_api_get.ts
const apiKey = "";
const companyId = "";
const apiVersion = "";

const headers = new Headers({
  "Authorization": `Bearer ${apiKey}`,
  "Content-Type": "application/json",
  "Accept": `application/json;version=${apiVersion}`,
});

const url = `https://cloudapi.inflowinventory.com/${companyId}/sales-orders`;
const response = await fetch(url, { headers });

console.log(response);
